internal/handlers/tigris/msg_find.go

```diff
@@ -76,7 +76,7 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 	}
 
 	var reply wire.OpMsg
-	must.NoError(reply.SetSections(wire.OpMsgSection{
+	if err := reply.SetSections(wire.OpMsgSection{
```

Comment:
This used to panic on `-0`.

Reply:
But `diff.md` says that in case of `-0` we return `0`. This code looks like we return an error.

Reply:
Since now it's handled by validation this doesn't need to change anymore. But I was surprised to see that updating DB with -0 was possible but it panic on trying to fetch value from DB. FYI, at the time I was implementing it, I didn't even know why negative zero was produced. :see_no_evil: I'll undo this change